src/rest_framework_dso/filters/__init__.py

```diff
@@ -82,7 +82,6 @@ class View(GenericAPIView):
 
 .. _django-filter: https://django-filter.readthedocs.io/
 """
-from . import lookups  # noqa (import is needed for registration)
```

Comment:
I think this whole file can go.

Reply:
indeed, thanks! this error happened as I was splitting up commits.